cap-ci/scripts/test.tmpl

```diff
@@ -2,8 +2,45 @@
 {{- /* Attention: PWD = ___/catapult here */}}
 {{- /* run test suites on cluster */}}
 {{- /* See: https://github.com/SUSE/catapult/wiki/Running-SCF-tests#kubecf */}}
-export KUBECF_TEST_SUITE="${KUBECF_TEST_SUITE:-smokes}"
-export KUBECF_NAMESPACE="scf"
-export QUIET_OUTPUT=true
-make tests-kubecf
+
+set -o errexit -o nounset -o pipefail
+
+if [[ "${KUBECF_TEST_SUITE}" == "mits" ]]; then
```

Comment:
This sounds bad... can't we have a `cap-ci/scripts/test_mits.tmpl`?
A world with less `if`s is a better world!

Reply:
https://github.com/SUSE/cap/pull/49/commits/7565891d8877174309127420f89b03cc51efc00d